Simplified/AxisService/AxisService.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AxisService.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+@objc
+protocol BookDownloadBroadcasting {
+  func failDownloadWithAlert(forBook book: NYPLBook)
```

Comment:
These methods are already present in NYPLMYBooksDownloadCenter so no extra changes are required to be added.